components/calendar/src/chinese.rs

```diff
@@ -277,17 +299,17 @@ impl Calendar for Chinese {
                 match ordinal {
                     1 => tinystr!(4, "und"), // this implies the leap month is < 1, which is impossible
                     2 => tinystr!(4, "und"), // this implies the leap month is = 1, which is impossible
-                    3 => tinystr!(4, "M02I"),
-                    4 => tinystr!(4, "M03I"),
-                    5 => tinystr!(4, "M04I"),
-                    6 => tinystr!(4, "M05I"),
-                    7 => tinystr!(4, "M06I"),
-                    8 => tinystr!(4, "M07I"),
-                    9 => tinystr!(4, "M08I"),
-                    10 => tinystr!(4, "M09I"),
-                    11 => tinystr!(4, "M10I"),
-                    12 => tinystr!(4, "M11I"),
-                    13 => tinystr!(4, "M12I"),
+                    3 => tinystr!(4, "M02"),
```

Comment:
nit: now that there are no I months, this match statement can be merged into the first one by doing an `if ordinal > leap month { ordinal -= 1 }` or something

Reply:
I think these should stay separate since the cases 1 and 2 are invalid for months after the leap month but valid for months before a leap month, and the case 13 is valid for after leap months but invalid for before leap months.

Reply:
Yeah but those can be handled in the debug assertions anyway, right? The `"und"` case should never be hit anyway.